### ISSUE TITLE
Link counter doesn't count clicks at the widget

### ISSUE THREAD

**Issue by SIRSteiner** [AmrDeveloper/LinkHub#28]
Link counter doesn't count clicks at the widget.

There is no more to explain ;-)

**Comment by AmrDeveloper** [comment]
Thank you for reporting @SIRSteiner

It solved :D

**Comment by SIRSteiner** [comment]
Sorry, with the latest changes nothing happens if you click at a link at the widget.

**Comment by AmrDeveloper** [comment]
@SIRSteiner You mean link is not open or counter is not updated, 

Can you try to remove the widget and add it again for testing

**Comment by SIRSteiner** [comment]
I mean link is not open also after removing widget. Tested in emulator Pixel 4 API 30

**Comment by AmrDeveloper** [comment]
That weird, i tested it on API 33, I got a similar problem until I opened chrome on browser and get started with it then it handle the intent and open link, btw I will try to test it on Pixel 4 API 30 and some real devices to find the problem

**Comment by SIRSteiner** [comment]
I dont't know why but now links are opened in Browser. I never had this issue at my smartphone just in emulator. Maybe only in emulator?

**Comment by AmrDeveloper** [comment]
Yes it happened only on emulators

**Comment by SIRSteiner** [comment]
Sorry but we can't close this issue because the counter is not updated immedially if the app is in background. Just after closing/quiting and reopen the app counters are updated. It's not a big problem but should be fixed ;-)

**Comment by AmrDeveloper** [comment]
> Sorry but we can't close this issue because the counter is not updated immedially if the app is in background. Just after closing/quiting and reopen the app counters are updated. It's not a big problem but should be fixed ;-)

You are right let's keep it and improve it after finishing other important issues
